cmd/dep/main.go

```diff
@@ -40,7 +40,28 @@ func main() {
 		&hashinCommand{},
 	}
 
+	examples := [][2]string{
+		{
+			"dep init",
+			"set up a new project",
+		},
+		{
+			"dep ensure",
+			"install the project's dependencies",
+		},
+		{
+			"dep ensure -update",
+			"update the saved versions of all dependencies",
```

Comment:
Let's use "locked" instead of "saved" here. It might not be as immediately familiar, but it's more precise, and the simple fact that there's a file with "lock" in the name is probably sufficient for confused folks to figure it out 😄